app/src/main/java/uk/ac/ebi/atlas/search/metadata/CellTypeWheelDao.java

```diff
@@ -23,6 +23,47 @@
 import static uk.ac.ebi.atlas.solr.cloud.collections.SingleCellAnalyticsCollectionProxy.ONTOLOGY_ANNOTATION_PART_OF_LABELS;
 import static uk.ac.ebi.atlas.solr.cloud.collections.SingleCellAnalyticsCollectionProxy.ONTOLOGY_ANNOTATION_SYNONYMS;
 
+// curl -u $SOLR_USER:$SOLR_PASS http://$SOLR_HOST/solr/scxa-analytics/query?rows=0 -d '
+// {
+//   "query": "ontology_annotation_ancestors_labels_t:\"lung\" OR ontology_annotation_parent_labels_t:\"lung\" OR ontology_annotation_part_of_rel_labels_t:\"lung\" OR ontology_annotation_synonyms_t:\"lung\" OR ontology_annotation_label_t:\"lung\"",
+//   "filter": "!ctw_cell_type:\"not applicable\" AND ctw_organism:\"Mus musculus\"",
+//   "facet": {
+//     "organisms": {
+//       "type": "terms",
+//       "field": "ctw_organism",
+//       "limit": -1,
+//       "facet": {
+//         "organismParts": {
+//           "type": "terms",
+//           "field": "ctw_organism_part",
+//           "limit": -1,
+//           "facet": {
+//             "organismParts": {
+//               "type": "terms",
+//               "field": "ctw_organism_part",
+//               "limit": -1,
+//               "facet": {
+//                 "cellTypes": {
+//                   "type": "terms",
+//                   "field": "ctw_cell_type",
+//                   "limit": -1,
+//                   "facet": {
+//                     "experimentAccessions": {
+//                       "type": "terms",
+//                       "field": "experiment_accession",
+//                       "limit": -1
+//                     }
+//                   }
+//                 }
+//               }
+//             }
+//           }
+//         }
+//       }
+//     }
+//   }
+// }
+// '
```

Comment:
If we need this comment here, maybe a short 1 sentence summary would be good at the top of these lines.